### PR TITLE
feat(site): add back benchmark to site (also change cpu model)

### PR DESCRIPTION
**Pull request by H01001000**
image

### CODE REVIEW

File: .github/workflows/site.yml [hunk 1]
```diff
@@ -6,23 +6,23 @@ name: Deploy Docs Site
 on:
   # Triggers the workflow on push or pull request events but only for the "main" branch
   push:
-    branches: ['main']
+    branches: ["main"]
```
Comment (itohatweb) [inline .github/workflows/site.yml:9]: Please undo those quote changes.

File: packages/benchmark/src/generateMessage.ts [hunk 2]
```diff
@@ -37,7 +37,7 @@ for (const benchmark of latestBaseBenchmarks.benches) {
     [latestBaseBenchmarks.commit.id]: benchmark,
   }
 }
-for (let i = 0; i < benchmarks.length; i++) {
+for (let i = benchmarks.length - 1; i > -1; i--) {
```
Comment (itohatweb) [inline packages/benchmark/src/generateMessage.ts:40]:
```suggestion
for (let i = benchmarks.length - 1; i >= 0; i--) {
```

File: scripts/generateMessage.js [hunk 3]
```diff
@@ -14,7 +14,7 @@ for (const benchmark of latestBaseBenchmarks.benches) {
     [latestBaseBenchmarks.commit.id]: benchmark,
   }
 }
-for (let i = 0; i < benchmarks.length; i++) {
+for (let i = benchmarks.length - 1; i > -1; i--) {
```
Comment (itohatweb) [inline scripts/generateMessage.js:17]:
```suggestion
for (let i = benchmarks.length - 1; i >= 0; i--) {
```

**Review by itohatweb — CHANGES_REQUESTED** [review]
Looks good, just a few minor changes.
Could you send a screenshot of how this would look like?

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**